decidim-initiatives/app/controllers/decidim/initiatives/admin/initiatives_controller.rb

```diff
@@ -183,6 +185,10 @@ def export_pdf_signatures
 
         private
 
+        def show_initiative_type_callout?
+          Decidim::InitiativesType.where(organization: current_organization).all.empty?
```

Comment:
using `none?` is easier, as is not loading all the records into memory.
 
```suggestion
          Decidim::InitiativesType.where(organization: current_organization).none?
```